doc/user/content/manage/reverse-etl/census.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Census"
+description: "How to use Census to export data out of Materialize."
+menu:
+  main:
+    parent: "reverse-etl"
+---
+
+This guide walks you through the steps required to create a [Census](https://www.getcensus.com/) sync using Materialize.
+
+## Before you begin
+
+In order to build a sync with Census you will need:
+
+* A table, view, materialized view or source within your Materialize account that you would like to export.
+* The connection details for one of [destinations](https://www.getcensus.com/integrations) supported by Census.
```

Comment:
Not sure if this would be better if it was more prescriptive and we just choose something common like google sheets.